etc/README.md

```diff
@@ -0,0 +1,46 @@
+# Installation instructions
+
+## bash
+
+Open your `.bashrc` file and add:
+
+> If you want to set up Git to automatically have Bash shell completion for all users, copy the `hub.bash_completion` script to the `/opt/local/etc/bash_completion.d` directory on Mac systems or to the `/etc/bash_completion.d/` directory on Linux systems. ([Source](https://git-scm.com/book/en/v1/Git-Basics-Tips-and-Tricks#Auto-Completion))
```

Comment:
Ah, got it. :)  I'm not sure on what to change, though, do you think it's better to just remove that whole snippet? Or should I add a h2 header above bash for Homebrew and mention that doing `brew install hub` automatically sets up autocompletion?